src/hash/rescue/rpo/mod.rs

```diff
@@ -52,6 +53,17 @@ mod tests;
 /// to deserialize them into field elements and then hash them using
 /// [hash_elements()](Rpo256::hash_elements) function rather then hashing the serialized bytes
 /// using [hash()](Rpo256::hash) function.
+///
+/// ## Domain separation
```

Comment:
Should we make similar comments for the [Rpx256](https://github.com/0xPolygonMiden/crypto/blob/50b5bf1d875e995ae129eec54d55b752a8586c37/src/hash/rescue/rpx/mod.rs#L59) struct?

Reply:
Add an adapted comment to RPX